<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWidget</class>
 <widget class="QFrame">
  <property name="minimumHeight">
   <number>16</number>
  </property>
  <property name="maximumHeight">
   <number>16</number>
  </property>
  <property name="frameStyle">
   <set>QFrame::NoFrame</set>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="openPushButton">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
